config/jobs.txt

```diff
@@ -12,12 +12,12 @@ Chief Medical Officer=1,1
 Assistant=-1,-1
 
 Quartermaster=1,1
-Cargo Technician=4,2
-Shaft Miner=3,3
+Cargo Technician=6,4
+Shaft Miner=4,4
```

Comment:
idk if this is in the active config, but at 4 this is broken, miners only support 3 people

Reply:
This is indeed what the live config looks like. I thought we made adjustments for this? We can lower it though.

Reply:
we definitely should, most maps only have 3 miner spawn points and this has resulted in miners spawning in wrong departments for a long time, however I haven't seen this happen recently. Weird, maybe less people want to play miner now 🤷